internal/pkg/fflag/fflag.go

```diff
@@ -0,0 +1,29 @@
+package fflag
+
+import (
+	"github.com/frain-dev/convoy/config"
+	"github.com/frain-dev/convoy/datastore"
+)
+
+var features = map[string]datastore.FlagType{}
+
+type FFlag struct{}
+
+func NewFFlag() *FFlag {
+	return &FFlag{}
+}
+
+func (c *FFlag) CanAccessFeature(key string, cfg *config.Configuration) bool {
```

Comment:
@danvixent The logic here is not correct. We should compare if the retrieved `flagType` is less than or equal to the user's selection.